README.md

```diff
@@ -154,6 +155,15 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 * [Superset](https://superset.incubator.apache.org/) - Modern, enterprise-ready business intelligence web application.
 * [Tableau](https://www.tableau.com) - Powerful and fastest growing data visualization tool used in the business intelligence industry.
 
+## Data-centric AI
+
+*Tools for improving training data to improve ML models.*
+
+* [cleanlab](https://github.com/cleanlab/cleanlab): Python library for data-centric AI and machine learning with messy, real-world data and labels.
+* [snorkel](https://github.com/snorkel-team/snorkel) - A system for quickly generating training data with weak supervision.
```

Comment:
`Snorkel` seems to fit in the `Data Enrichment` section, as this is related to labeling and data augmentation.

Also, could you apply capitalization please?

```suggestion
* [Snorkel](https://github.com/snorkel-team/snorkel) - A system for quickly generating training data with weak supervision.
```

Reply:
applied caps and moved it to Data Enrichment section